packages/salesforce/data_stream/login/fields/fields.yml

```
@@ -0,0 +1,118 @@
+- name: salesforce
+  type: group
+  fields:
+    - name: instance_url
+      type: keyword
+      description: The Instance URL of the Salesforce instance.
+    - name: login
+      type: group
+      fields:
+        - name: application
+          type: keyword
+          description: The application used to access the Salesforce organization.
+        - name: api
+          type: group
+          fields:
+            - name: type
+              type: keyword
+              description: The type of Salesforce API request.
+            - name: version
+              type: keyword
+              description: The version of the Salesforce API used for the login request.
+        - name: auth
+          type: group
+          fields:
+            - name: service_id
+              type: keyword
+              description: The authentication service ID used by a third-party identity provider for single sign-on (SSO) using the OpenID Connect protocol.
+            - name: method_reference
+              type: keyword
+              description: The authentication method reference used by a third-party identity provider for SSO using the OpenID Connect protocol. This field is available in API version 51.0 and later.
+        - name: key
+          type: keyword
+          description: The string that ties together all events in a given user's login session, starting with the login event and ending with either a logout event or the user session expiring.
+        - name: history_id
+          type: keyword
+          description: The identifier that tracks a user session, allowing correlation of user activity with a specific login instance.
+        - name: type
+          type: keyword
+          description: The type of login used to access the session.
+        - name: geo_id
+          type: keyword
+          description: The Salesforce ID of the LoginGeo object associated with the user's IP address during login.
+        - name: client
+          type: group
+          fields:
+            - name: ip
+              type: ip
+              description: The IP address of the client using Salesforce services. Internal Salesforce IP addresses are shown as "Salesforce.com IP".
+        - name: cpu_time
+          type: long
+          description: The CPU time in milliseconds used to complete the login request, indicating the amount of activity in the application server layer.
+          unit: ms
+          metric_type: gauge
+        - name: db_total_time
+          type: double
+          description: The total time in milliseconds for the database round trip during login, including time spent in the JDBC driver, network, and database CPU time.
+          unit: ms
```

```suggestion
        - name: db_total_time
          type: long
          description: The time in nanoseconds for the database round trip during login, including time spent in the JDBC driver, network, and database CPU time.
          unit: nanos
```

According to [link](https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_eventlogfile_login.htm#:~:text=app%20server%20layer.-,DB_TOTAL_TIME,-Type)